api/src/tests/test_data/multiflash_input

```diff
@@ -0,0 +1 @@
+{"number_of_components": 24, "component_ids": [3, 2, 1, 101, 201, 301, 401, 402, 503, 504, 605, 701, 801, 901, 1016, 1301, 1501, 1701, 2001, 2301, 2601, 3101, 3901, 5], "temperature": 20, "pressure": 21, "composition": [0.173798348, 0.0039, 0.0247, 0.514984565, 0.067070685, 0.0419, 0.0085, 0.0181, 0.00742, 0.00907, 0.011647523, 0.0187, 0.0198, 0.0127, 0.016, 0.0122, 0.00933, 0.00713, 0.00544, 0.00733, 0.00427, 0.00365, 0.00232, 0.01]}
```

Comment:
could we convert these to ether .py og .json files?
Would help on readability

Reply:
Yep, will do.